app/states/testMulti.js

```diff
@@ -0,0 +1,82 @@
+import StackQuest from '../main'
+
+// Util functions for firebase database
+import {addCharacter, GamePlayers, onCharacterUpdate, updateCoordinates, updateOnCharacterMovement} from './firebase-database'
+const dummyCharId = `character${Math.floor(Math.random()*50)}`
+
+let map
+  , layer
+  , cursors
+  , fixedText
+
+const testState = {
+  preload() {
+    StackQuest.load.spritesheet('link', '../assets/link.png', 100, 50)
+    StackQuest.load.tilemap('testmap', 'maps/testmap.json', null, Phaser.Tilemap.TILED_JSON)
+    StackQuest.load.image('tiles', 'maps/terrain.png')
+  },
+
+  create() {
+    StackQuest.physics.startSystem(Phaser.Physics.P2JS)
+
+    map = StackQuest.add.tilemap('testmap')
+
+    map.addTilesetImage('terrain', 'tiles')
+
+    layer = map.createLayer('terrainlayer')
+
+    layer.resizeWorld()
+
+    map.setCollisionBetween(278, 280)
+    map.setCollision(310)
+    map.setCollision(312)
+    map.setCollisionBetween(342, 344)
+
+    StackQuest.physics.p2.convertTilemap(map, layer)
```

Comment:
references to StackQuest can be changes to this. This refers to the state object the file is exporting, and it has access to the physics, camera, etc. objects that StackQuest does.